src/org/omegat/core/Core.java

```diff
@@ -258,6 +260,7 @@ public static void initializeGUI(ClassLoader classLoader, final Map<String, Stri
         filterMaster = new FilterMaster(Preferences.getFilters());
 
         // 4. Initialize other components. They add themselves to the main window.
+        URLProtocolHandler.install();
```

Comment:
It seems like it would be nice to initialize this package more like an OmegaT plugin: have a class that implements `loadPlugins` and `unloadPlugins`. That way the core doesn't need to know about it.

Reply:
I'd like to try.

Reply:
Aah, get it. I can implement it as in-source plugin.